server/sonar-server/src/main/java/org/sonar/server/search/Result.java

```diff
@@ -75,31 +72,52 @@ public Result(@Nullable BaseIndex<K, ?, ?> index, SearchResponse response) {
 
   private void processAggregation(Aggregation aggregation) {
```

Comment:
I'm not sure that all aggregation stuff should be put in this class : why not having a kind of helper class that will take a SearchResponse in parameter and add process aggregation on it ?